### DISCUSSION TITLE
Modrinth?

### DISCUSSION THREAD

**Discussion by ghost** [SkriptLang/Skript#6213] [Suggestions]
What about uploading Skript to [Modrinth](https://modrinth.com)?

**Comment by ScienceKit** [comment]
I mean Modrinth is mostly for mods, but it does have options for most MC-related content. I want this.

**Reply by ghost** [reply]
It has support for plugins as well.

**Comment by ghost** [comment]
Modrinth actually pays people, It's not like other websites that shove ads up your throat, It's just one small text and logo ad above the content you're looking at.

Another thing, Aternos may actually add Skript to their plugin catalog, since they allow plugins from Modrinth to be **suggested**
